page-content/Changes.html

```diff
@@ -11,7 +11,7 @@ <h3 class="page-title">Submit Changes</h3>
                 <span class="input-group-text">Type</span>
             </div>
             <select class="custom-select" id="change-type-go-input" aria-label="Change type select with go button">
-                <option selected disabled>Choose...</option>
+                <!--<option selected disabled>Choose...</option>-->

```

Comment:
Why remove this one?

Reply:
I thought it would be cleaner not to have an option labelled choose. I can add this back if you want me to.

Reply:
We can leave it out, but if you want to remove it, simply delete it.

Reply:
Ok sure I guess I will delete it then